roles/config-lvm/tasks/lvm.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+- name: "Setup and create PV & VG"
+  lvg:
+    vg: "{{ item.vg_name }}"
+    pvs: "{{ item.storage_device }}"
+
+- name: "Setup LV"
+  lvol: 
+    vg: "{{ item.vg_name }}"
+    lv: "{{ item.lv_name }}"
+    size: "100%VG"
+
+- name: "Create file system on share"
+  shell: "mkfs.xfs -f /dev/mapper/{{ item.vg_name }}-{{ item.lv_name }}"
```

Comment:
can this use the ansible module:  http://docs.ansible.com/ansible/latest/filesystem_module.html

Reply:
done